packages/sdk/vite.config.ts

```diff
@@ -10,7 +10,10 @@ export default defineConfig({
     lib: {
       entry: {
         index: resolve(__dirname, "src/index.ts"),
-        "browser-wallets": resolve(__dirname, "src/browser-wallets/index.ts"),
+        "browser-wallets-unisat": resolve(
+          __dirname,
+          "src/browser-wallets/unisat/index.ts",
+        ),
```

Comment:
Explicit exports are used for treeshaking so that the bundler doesn't import all dependencies in one object which can be expensive as more functionality is added to the SDK.

There is definitely some overhead in this setup which I am not keen about but I found this to be the most optimal approach so far.